ChainReactionBindings/TestBase/Pages/WishlistPage.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Runtime.InteropServices;
+using OpenQA.Selenium;
+
+namespace ChainReactionBindings.TestBase.Pages
+{
+	class WishlistPage: BasePage
+	{
+		public string WishlistHeaderId = ".user_profile_headding2";

```

Comment:
Please make all the ids private, we should not give access to them outside the PageObject